modules/multi-arch-creating-podplacment-config-using-cli.adoc

```diff
@@ -0,0 +1,58 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-creating-podplacement-config-using-cli_{context}"]
+= Creating a ClusterPodPlacementConfig CR by using the CLI
+
+You can create a `ClusterPodPlacementConfig` custom resource (CR) by using the {oc-first}.
```

Comment:
```suggestion
You can create a `ClusterPodPlacementConfig` object by using the {oc-first}.
```

How can we also highlight that this must be named `cluster`?

Reply:
Added a footnote with the following info:

"<1> Set this field value to `cluster`. You must set the name of the `ClusterPodPlacementConfig` object to `cluster`. Otherwise, the `ClusterPodPlacementConfig` object creation fails. "